src/main/java/uk/gov/gchq/palisade/rule/Rules.java

```diff
@@ -38,19 +37,20 @@
  * @param <T> The type of data records that the rules will be applied to.
  */
 @JsonPropertyOrder(value = {"message", "rules"}, alphabetic = true)
-public class Rules<T> {
+public class Rules<T extends Serializable> implements Serializable {
+    private static final long serialVersionUID = 1L;
     private static final String ID_CANNOT_BE_NULL = "The id field can not be null.";
     private static final String RULE_CANNOT_BE_NULL = "The rule can not be null.";
     public static final String NO_RULES_SET = "no rules set";
 
     private String message;
-    private Map<String, Rule<T>> rulesHashMap;
+    private HashMap<String, Rule<T>> rulesMap;
```

Comment:
It is important to maintain order of the rules so they are applied in the order they are inserted into the map.
```suggestion
    private LinkedHashMap<String, Rule<T>> rulesMap;
```